bfdd/ptm_adapter.c

```diff
@@ -128,6 +128,35 @@ static void debug_printbpc(const struct bfd_peer_cfg *bpc, const char *fmt, ...)
 		   timers[0], timers[1], timers[2], cbit_str);
 }
 
+static void
+_ptm_bfd_session_del(struct bfd_session *bs, uint8_t diag)
+{
+	if (bglobal.debug_peer_event)
+		zlog_debug("session-delete: %s", bs_to_string(bs));
```

Comment:
shouldn't we use `__func__`?

Reply:
unfortunately no. The BFD functions got too split and to keep track of what is doing what I created some convention to make these log messages different (somewhat like `rt_netlink.c` has) but grouped in sections (in this case `session-delete`, there is more of this session in `bfd.c`).

Also I don't like that many of the functions we've got from ptm-bfd have long names, but we'd need a refactor to get these fixed (and a lot of creativity to come up with names).

Reply:
:+1: